src/components/progress-tracker/progress-tracker.test.js

```diff
@@ -28,15 +28,27 @@ context("Tests for ProgressTracker component", () => {
       "aria-valuetext",
     ])("when %s prop is passed", (propName) => {
       it(`verify that the ${propName} is set to cypress-standard`, () => {
-        const props = { [propName]: CHARACTERS.STANDARD };
+        let props;
+        if (
+          !propName.endsWith("now") &&
```

Comment:
You're checking this condition twice might be worth storing it in a variable

Reply:
As you've also done the same check below maybe move it to a function but I'm happy for you to just go with with just storing it in constants if you believe the below isn't worth the effort etc

```
// globally in file
const DEFAULT_PROP_VALUE = 50;

const checkPropName = (propName) => propName.endsWith("now") && propName.endsWith("min") && propName.endsWith("max");

const getProps = (propName, shouldBeDefault) => {
if (shouldBeDefault) {
  return { [propName]: DEFAULT_PROP_VALUE };
}
  return { [propName]: CHARACTERS.STANDARD };
}


// inside test cases 
const isNowMinOrMax = checkPropName(propName);

const props = getProps(propName, isNowMinOrMax);

const assertion = isNowMinOrMax ? DEFAULT_PROP_VALUE : CHARACTERS.STANDARD;

...
```